extensions/BeatBlox/index.js

```diff
@@ -220,7 +220,7 @@
 
             if(note === 'Rest') return availableNotes[note];
 
-            if(!isNaN(Number(note))) return Number(note);
+            if(Number.isInteger(Number(note))) return note;
```

Comment:
Well this actually removed the parse for the return value. You just need to hold on to the intermediate.

The only other issue I found is that the `Number` parser treats regex `\s*` as 0, which we probably don't want either (i.e. if the student forgets to put a value in the note slot it'll just play a ridiculously low note).